web/portal/src/main/webapp/WEB-INF/conf/sso/oauth-configuration.xml

```diff
@@ -155,21 +155,13 @@
         <name>accessType</name>
         <value>${exo.oauth.openid.accessType}</value>
       </value-param>
-      <value-param>
-        <name>authenticationURL</name>
-        <value>${exo.oauth.openid.authenticationURL}</value>
-      </value-param>
-      <value-param>
-        <name>accessTokenURL</name>
-        <value>${exo.oauth.openid.accessTokenURL}</value>
-      </value-param>
       <value-param>
         <name>tokenInfoURL</name>
         <value>${exo.oauth.openid.tokenInfoURL}</value>
       </value-param>
       <value-param>
```

Comment:
This should be removed also

Reply:
how i can get  tokenInfoURL  is not exist in .well-known/openid-configuration

Reply:
TokenInfoUrl is used to validate the token.
As we change this to validate the token locally, we dont need it.
In addition, this url is not provided by all idp vendors